doc: add ppp 7 (#456)
itchysats/10101: docs/ppp/iteration-7-30112022.md

### Plans 📆

> We are pleased that the judges found our project worthy and are pumped to get one more week of hacking in before the final pitch day!

Reviewer: ```suggestion
We are excited that the judges found our project worthy and are pumped to get one more week of hacking in before the final pitch day!
```

> We are pleased that the judges found our project worthy and are pumped to get one more week of hacking in before the final pitch day!

Reviewer 2: Just pleased?!?

```suggestion
We are thrilled that the judges have deemed our project worthy of the final round and are pumped to get one more week of hacking in before the final pitch day!
```

For this, we decided to mostly work on UX improvements and get our App into Apple's Testflight and an app store for Android.

A few things on our todo list for this week are:

> - Persist lightning payment and bitcoin transaction history

Reviewer 2: ```suggestion
- Persist Lightning payment and Bitcoin transaction history
```

> - Small UI changes

Reviewer: ```suggestion
- UI improvements & polishing
```

- Simplify on-boarding by funding the app through our own faucet
- Show transaction details for Lightning as well as for Bitcoin
- Release version 0.3.0 for Testflight and Android

The main goal of this iteration is to get a usable version out there to on-board early users and collect feedback.

> If you are keen, feel free to reach out.

Reviewer: ```suggestion
If you are keen, please reach out!
```

### Progress ✅

Last week was probably one of our most productive ones. Kudos to the whole team.

- Published a blog post about our [vision](https://makers.bolt.fun/story/10101-pitch--441)
- Created a Pitch video [https://rumble.com/v1wkrpc-10101-legends-of-lightning-pitch.html](https://rumble.com/v1wkrpc-10101-legends-of-lightning-pitch.html)

> - Had a public demo with the community. Recording can be found [here](https://rumble.com/v1wr04i-10101-demo.html)

Reviewer: ```suggestion
- Had a public demo with the community. Recording can be found [here](https://rumble.com/v1wr04i-10101-demo.html), demo summary post can be found [here](https://makers.bolt.fun/story/10101-its-demo-time--470)
```

> - Fix heaps of bugs

Reviewer 2: 🔧 To keep the tense consistent.

```suggestion
- Fixed heaps of bugs
```

- Released version 0.2.1 as an APK downloadable from our [GitHub repository](https://github.com/itchysats/10101/releases)
- Wired up CFD settlement end-to-end

### Problems ✋

> > "Problems are like washing machines. They twist us, spin us and knock us around. But in the end we come out cleaner, brighter and better than before.", Unknown

Reviewer 2: Hahaha, so good.
